src/lib/support/logging/CHIPLogging.h

```diff
@@ -296,5 +296,42 @@ bool IsCategoryEnabled(uint8_t category);
 
 #endif // CHIP_CONFIG_ENABLE_FUNCT_ERROR_LOGGING
 
+/*  @def ChipLogFormatX64
+ *
+ *  @brief
+ * 	Macro for use in a string formatter for a 64-bit hex print.
+ *      Will split into 2x 32-bit prints to support small printf libraries
+ *
+ *  Example Usage:
+ *
+ *  @code
+ *  void foo() {
+ *      uint64_t value = 0x1122334455667788;
```

Comment:
suprenit: indent here seems a bit off

Reply:
seemed to be a stray tab. new editor wasn't set correctly - clang-format doesn't catch this it seems ?